src/tutorials/0005-regular-files-api/03.js

```diff
@@ -38,15 +49,16 @@ const validate = async (result, ipfs) => {
   if (JSON.stringify(expectedResult) === JSON.stringify(result)) {
     return {
       success: 'Success! You did it!',
-      logDesc: 'Your `add` command returned the array of objects below. Notice in particular the `hash` ' + valueText + ", since we'll need " + thatText + ' to access ' + fileText + ' again later. The `path` matches the `hash` for ' + fileText + ", but we'll see in future lessons that that's not always true.",
-      log: result
+      logDesc: 'Your `add` command returned the array of objects below. Notice in particular the `cid` ' + valueText + ", since we'll need " + thatText + ' to access ' + fileText + ' again later. The `path` matches the `cid` for ' + fileText + ", but we'll see in future lessons that that's not always true.",
```

Comment:
Probably really matching a field inside the shrunken CID object

Reply:
Even if it's that field, it still represents the CID, so I don't think we need to distinguish here in this case.